src/components/Amalgamation/BusinessTableSummary.vue

```diff
@@ -1,63 +1,80 @@
 <template>
-  <v-simple-table id="business-table-summary">
-    <template #default>
-      <thead>
-        <tr>
-          <th>Business Name</th>
-          <th>Mailing Address</th>
-          <th>Role</th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <tr v-if="!getAmalgamatingBusinesses.length">
-          <td colspan="6">
-            <p class="text-center mb-0">
-              No businesses added
-            </p>
-          </td>
-        </tr>
-
-        <tr
-          v-for="item in getAmalgamatingBusinesses"
-          :key="key(item)"
-        >
-          <td class="business-name">
-            <v-icon color="gray9">
-              mdi-domain
-            </v-icon>
-            <strong>{{ name(item) }}</strong><br>{{ email(item) }}
-          </td>
-
-          <td class="business-address">
-            <template v-if="item.type === AmlTypes.LEAR">
-              <BaseAddress
-                v-if="item.address"
-                :address="item.address"
-              />
-              <span v-else>Affiliate to view</span>
-            </template>
-
-            <template v-if="item.type === AmlTypes.FOREIGN">
-              {{ jurisdiction(item) }}
-            </template>
-          </td>
-
-          <td class="business-role">
-            {{ role(item) }}
-          </td>
-        </tr>
-      </tbody>
-    </template>
-  </v-simple-table>
+  <section>
+    <div
+      v-if="!getAmalgamatingBusinessesValid"
+      class="defineCompanyStepErrorMessage"
+    >
+      <span>
+        <v-icon color="error">mdi-information-outline</v-icon>
+        <span class="error-text mx-1">This step is unfinished.</span>
+        <span>
+          <router-link
+            v-if="!getAmalgamatingBusinessesValid"
+            :to="{ path: `/${RouteNames.AMALG_REG_INFORMATION}` }"
```

Comment:
![step is unfinished amalg table](https://github.com/bcgov/business-create-ui/assets/122301442/f5340adc-ad60-47c3-a99e-952367ba9de7)

Reply:
Did Yui ask for this? I discussed this with him previously and was OK with the two conditional router links in SummaryDefineCompany.vue.

Reply:
I didn't. I just thought we need it. Sorry about that. I'll be reverting this change then.

Reply:
Reverted this change.